--- origsrc/libgdiplus-1.9/configure.in	2008-01-29 16:00:28.000000000 -0600
+++ src/libgdiplus-1.9/configure.in	2008-04-22 14:41:10.796875000 -0500
@@ -32,7 +32,6 @@
 	# disable the PNG, PDF and SVG surface backends of Cairo (unrequired for GDI+)
 	ac_configure_args="$ac_configure_args --disable-png --disable-pdf --disable-svg"
 
-	AC_CONFIG_SUBDIRS(cairo)
 	CAIRO_DIR="cairo"
 	CAIRO_LIBS='$(top_builddir)/cairo/src/libcairo.la'
 	CAIRO_CFLAGS='-I$(top_builddir)/cairo/pixman/src -I$(top_srcdir)/cairo/pixman/src -I$(top_builddir)/cairo/src -I$(top_srcdir)/cairo/src'
@@ -194,7 +193,7 @@
 case "$host" in
 	i*86-*-*)
 		case $host_os in
-		  solaris*)
+		  solaris*|cygwin*)
 			# On solaris 10 x86, gcc prints a warning saying 'visibility attribute not supported on this configuration; ignored', but linking fails.
 			have_visibility_hidden=no
 		esac
--- origsrc/libgdiplus-1.9/src/Makefile.am	2008-01-29 16:00:28.000000000 -0600
+++ src/libgdiplus-1.9/src/Makefile.am	2008-04-22 10:52:37.203125000 -0500
@@ -118,6 +118,8 @@
 	wmfcodec.c			\
 	wmfcodec.h
 
+libgdiplus_la_LDFLAGS = -avoid-version -no-undefined
+
 libgdiplus_la_LIBADD = $(GDIPLUS_LIBS)
 
 INCLUDES = $(GDIPLUS_CFLAGS) -Wall -Wno-unused -Wno-format
